code/modules/admin/adminhelp.dm

```diff
@@ -223,22 +223,24 @@
 
 	var/audio
 
-	for (var/client/C)
-		if (!C.mob) continue
+	for (var/client/C as anything in global.clients)
+		if (!C.holder)
+			continue
+		if (!C.mob)
+			continue
 		var/mob/M = C.mob
-		if (C.holder)
-			if (!M.client.holder.hear_prayers || (M.client.player_mode == 1 && M.client.player_mode_ahelp == 0)) //XOR for admin prayer setting and player mode w/ no ahelps
-				continue
-			else
-				boutput(M, "<span class='notice' [in_chapel? "style='font-size:1.1em'":""]><B>PRAYER: [is_atheist ? "(ATHEIST) " : ""]</B><a href='?src=\ref[M.client.holder];action=subtlemsg&targetckey=[client.ckey]'>[client.key]</a> / [client.mob.real_name ? client.mob.real_name : client.mob.name] <A HREF='?src=\ref[M.client.holder];action=adminplayeropts;targetckey=[client.ckey]' class='popt'><i class='icon-info-sign' />: <I>[msg]</I></span>")
-				if(M.client.holder.audible_prayers == 1)
-					M << sound("sound/misc/boing/[rand(1,6)].ogg", volume=50, wait=0)
-				else if(M.client.holder.audible_prayers == 2) // this is a terrible idea
-					if(!audio)
-						audio = dectalk(msg)
-					var/vol = M.client.getVolume(VOLUME_CHANNEL_ADMIN)
-					if(vol)
-						M.client.chatOutput.playDectalk(audio["audio"], "prayer by [src] ([src.ckey]) to [M.ckey]", vol)
+		if (!M.client.holder.hear_prayers || (M.client.player_mode == 1 && M.client.player_mode_ahelp == 0)) //XOR for admin prayer setting and player mode w/ no ahelps
+			continue
+		else
+			boutput(M, "<span class='notice' [in_chapel? "style='font-size:1.1em'":""]><B>PRAYER: [is_atheist ? "(ATHEIST) " : ""]</B><a href='?src=\ref[M.client.holder];action=subtlemsg&targetckey=[client.ckey]'>[client.key]</a> / [client.mob.real_name ? client.mob.real_name : client.mob.name] <A HREF='?src=\ref[M.client.holder];action=adminplayeropts;targetckey=[client.ckey]' class='popt'><i class='fas fa-circle-info' />: <I>[msg]</I></span>")
```

Comment:
```suggestion
			boutput(M, "<span class='notice' [in_chapel? "style='font-size:1.1em'":""]><B>PRAYER: [is_atheist ? "(ATHEIST) " : ""]</B><a href='?src=\ref[M.client.holder];action=subtlemsg&targetckey=[client.ckey]'>[client.key]</a> / [client.mob.real_name ? client.mob.real_name : client.mob.name] <A HREF='?src=\ref[M.client.holder];action=adminplayeropts;targetckey=[client.ckey]' class='popt'><i class='fas fa-circle-info'></i>: <I>[msg]</I></span>")
```
this doesn't close properly and causes prayers to be ugly and allcaps as they are in Font Awesome 6 font or whatever